internal/storage/v2/clickhouse/tracestore/dbmodel/from.go

```diff
@@ -41,10 +41,18 @@ func FromRow(storedSpan *SpanRow) ptrace.Traces {
 	return trace
 }
 
-func convertResource(sr *SpanRow) pcommon.Resource {
+func convertResource(sr *SpanRow, spanForWarnings ptrace.Span) pcommon.Resource {
 	resource := ptrace.NewResourceSpans().Resource()
 	resource.Attributes().PutStr(otelsemconv.ServiceNameKey, sr.ServiceName)
-	// TODO: populate attributes
+	putAttributes(
+		resource.Attributes(),
+		spanForWarnings,
```

Comment:
@yurishkuro does it make sense to add the warnings for resource attributes to the span?